### PR TITLE
Add latest update to master branch (Wlv 2016, nano v4 2016, nano v5 2017)

### PR DESCRIPTION
**Pull request by GLP90**

### CODE REVIEW

File: python/WlvHbb13TeV2016config/s.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+tr -d '\r' < cuts.ini                    > cuts.ini
```
Comment (perrozzi) [inline python/WlvHbb13TeV2016config/s.sh]: is this file needed?

File: python/WlvHbb13TeV2016config/volatile.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # this file has been created automatically and will be overwritten by submit.py!
 [Directories]
-plotpath: /mnt/t3nfs01/data01/shome/gaperrin/VHbb/CMSSW_7_4_3/src/Xbb/python/logs_v25_Wlv//sysnew-TEST5/Plots/
-logpath: /mnt/t3nfs01/data01/shome/gaperrin/VHbb/CMSSW_7_4_3/src/Xbb/python/logs_v25_Wlv//sysnew-TEST5/Logs/
```
Comment (perrozzi) [inline python/WlvHbb13TeV2016config/volatile.ini]: this should not be committed, I think

**Comment by perrozzi** [comment]
there are several modifications to script that have nothing to do with Wlnu, is this intended?

**Comment by perrozzi** [comment]
are you sure you want to remove all the txt files?

**Comment by perrozzi** [comment]
I guess we should change name to this PR, right? :)

File: python/WlvHbb13TeV2016config/volatile.ini (hunk 2 above)
Comment (GLP90) [reply python/WlvHbb13TeV2016config/volatile.ini]: This is ok (the path is overwritten at each job submission)

Comment (piberger) [reply python/WlvHbb13TeV2016config/volatile.ini]: the file itself should not be committed, since it will always have changes, that's why the automatically generated lines are in a separate file

Comment (GLP90) [reply python/WlvHbb13TeV2016config/volatile.ini]: sorry I didn't notice it was the volatile.ini file. It is now removed.

**Comment by GLP90** [comment]
I went through the files (except the .txt for all the samples) and it seems ok to merge.
